CHANGELOG.md

```diff
@@ -77,5 +78,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Fix an issue where sidecar fails when not run as root user.
 
-[Unreleased]: https://github.com/cyberark/conjur-authn-k8s-client/compare/v0.13.0...HEAD
```

Comment:
why are we bumping so many versions? 

from v0.13.0 -17?

Reply:
we are bumping only one version but i saw that the comparison section wasn't maintained. Please review this PR via the commits, it will make more sense.